Sources/VLCMovieViewController.m

```diff
@@ -450,6 +480,21 @@ - (void)viewWillAppear:(BOOL)animated
     }
 }
 
+- (void) layoutFrameForMultiSelectionView
+{
+    CGRect controllerPanelFrame = _controllerPanel.frame;
+    CGRect multiSelectionFrame;
+    
+    BOOL isLandscape = UIInterfaceOrientationIsLandscape([[UIApplication sharedApplication] statusBarOrientation]);
+    BOOL isIphone = (UI_USER_INTERFACE_IDIOM() == UIUserInterfaceIdiomPhone);
+    _multiSelectionView.showsEqualizer = isLandscape && isIphone;
```

Comment:
This breaks the current interface and is not the same logic as before.
the equalizer was always available on iPad.